Add locale es_PA / Spanish (Panama)

2ndkauboy opened the pull request:
## What?
Add the missing locale es_PA for Spanish spoken in Panama.

2ndkauboy committed:
Add locale es_PA / Spanish (Panama)

amieiro commented on locales/locales.php:
```diff
@@ -902,6 +902,18 @@ public function __construct() {
+		$es_pa->lang_code_iso_639_2 = 'spa';
+		$es_pa->lang_code_iso_639_3 = 'spa';
+		$es_pa->country_code = 'pa';
+		$es_pa->wp_locale = 'es_PA';
```
```suggestion
```
This should only be set for locales that are officially supported on translate.wordpress.org.

amieiro committed:
Remove the wp_locale

This should only be set for locales that are officially supported on translate.wordpress.org.

amieiro committed:
Merge branch 'develop' into locale-es-pa
